util/topgen.py

```diff
@@ -505,6 +508,7 @@ def generate_flash(topcfg: Dict[str, object], out_path: Path) -> None:
         "infos_per_bank": [10, 1, 2]
     })
 
+    params.pop('base_addrs', None)
```

Comment:
I assume this was expected to come from line 503. Is it really necessary to pop this? Seems confusing.